core/api/src/domain/notifications/index.ts

```diff
@@ -17,11 +17,25 @@ export const NotificationType = {
   LigtningPayment: "lightning_payment",
 } as const
 
+export const NotificationCategory = {
+  Payments: "Payments",
+  Circles: "Circles",
+  Price: "Price",
+  AdminNotification: "AdminNotification",
+  Balance: "Balance",
+  Marketing: "Marketing",
+} as const
+
 export const checkedToNotificationCategory = (
   notificationCategory: string,
 ): NotificationCategory | ValidationError => {
   // TODO: add validation
-  if (!notificationCategory) {
+  if (
+    !notificationCategory ||
+    !Object.values(NotificationCategory).find(
+      (category) => category === notificationCategory,
+    )
+  ) {
     return new InvalidPushNotificationSettingError("Invalid notification category")
```

Comment:
Maybe this is no more `InvalidPushNotificationSettingError`, probably require an specific error

Reply:
Added a more specific error